### ISSUE TITLE
Run builds in parallel

### ISSUE THREAD

**Issue by dgageot** [GoogleContainerTools/skaffold#248]
On initial build and thereafter, it should be possible to build all the artifacts in parallel.

**Comment by dgageot** [comment]
Let's close this one. All builders except the local one now build in parallel.

**Comment by tmc** [comment]
Should there be another issue for local parallelism?
